hs/src/Reach/APICut.hs

```diff
@@ -137,6 +150,11 @@ interactIn = interactX "in"
 interactOut :: DLExpr -> Bool
 interactOut = interactX "out"
 
+setApiDetails :: SLPart -> DLExpr -> Bool
+setApiDetails who = \case
```

Comment:
I think `is_setApiDetails` or something is better... the word `set` feels like it is an effect